recipes/lofreq/meta.yaml

```diff
@@ -1,21 +1,19 @@
+{% set version="2.1.4" %}
+
 package:
   name: lofreq
-  version: '2.1.3.1'
+  version: {{ version }}
 source:
-  url: https://github.com/CSB5/lofreq/raw/master/dist/lofreq_star-2.1.3.1_linux-x86-64.tgz # [linux]
-  md5: f987f568f44149a3d2b59a21db721cd7 # [linux]
-  url: https://github.com/CSB5/lofreq/raw/master/dist/lofreq_star-2.1.3.1_macosx.tgz # [osx]
-  md5: 4b8a7459841ba8b22b2f64a2dd18bef2 # [osx]
+  url: https://raw.githubusercontent.com/CSB5/lofreq/v{{ version }}/dist/lofreq_star-{{ version }}.tar.gz
+  sha256: e0c52ef01ded39e7c2440b097cce11ae278df23d9fca288051e21728b281f082
 build:
-  noarch: python
-  number: 1
-  script: "mkdir -p $PREFIX/bin; chmod a+x bin/*; cp bin/* $PREFIX/bin"
-  skip: False
+  number: 0
 requirements:
   build:
     - {{ compiler('cxx') }}
   host:
     - python
+    - htslib
```

Comment:
Is htslib being statically compiled in? If not it will be needed during runtime.

Reply:
Ah, apparently we changed the htslib recipe to auto export itself, that's nice.